lib/PRC/Schema/Result/User.pm

```diff
@@ -356,5 +358,19 @@ sub available_repos {
   })->all;
 }
 
+
+=head2 make_admin
```

Comment:
Let's not do the init_admin thing. I'd rather manually assign admin roles for now.